test/test-creation.js

```diff
@@ -54,6 +57,96 @@ describe("ESLint Rule Generator", function() {
             done();
         });
     });
+
+    describe("With pathological input", function() {
+        describe("Double quotes in description", function() {
+            beforeEach(function(done) {
+                helpers.mockPrompt(this.rule, {
+                    userName: "",
+                    ruleId: "test-rule",
+                    desc: "My \"foo\"",
+                    invalidCode: "var x = \"foo\";",
+                    target: "eslint"
+                });
+                this.rule.options["skip-install"] = true;
+                this.rule.run(done);
+            });
+
+            describe("Resulting rule file", function() {
+                beforeEach(function() {
+                    this.resultRuleModule = require(path.join(testDirectory, "lib", "rules", "test-rule"));
+                });
+
+                it("should be requireable", function() {
+                    assert.ok(this.resultRuleModule);
+                });
+
+                it("should have correct description", function() {
+                    assert.strictEqual(this.resultRuleModule.meta.docs.description, "My \"foo\"");
+                });
+            });
+        });
+
+        /*
+         * TODO: (platinumazure) Uncomment when eslint/generator-eslint#44 is
```

Comment:
Why not just fix #44 with this PR? I'm not a huge fan of checking in commented out code.